how-to/scale/with-ansible.md

```diff
@@ -480,16 +494,21 @@ For example, if you want to remove a PD node (node2) with the IP address `172.16
     | node8 | 172.16.10.8 | TiKV3 |
     | node9 | 172.16.10.9 | TiKV4 |
 
-5. Perform a rolling update to the entire TiDB cluster:
+5. Update the cluster configuration:
 
     ```
-    ansible-playbook rolling_update.yml
+    ansible-playbook deploy.yml
     ```
 
-6. Update the Prometheus configuration and restart the cluster:
+6. Restart the Prometheus, and disable the monitoring of Pd nodes used for increasing the capacity:
 
     ```
-    ansible-playbook rolling_update_monitor.yml --tags=prometheus
+    ansible-playbook stop.yml --tags=prometheus
+    ansible-playbook start.yml --tags=prometheus
     ```
 
 7. To monitor the status of the entire cluster, open a browser to access the monitoring platform: `http://172.16.10.3:3000`.
+
+> **Note:**
+>
+> The PD Client in TiKV caches PD node list. The list is updated only if the PD leader is switched or the TiKV is restarted to load the latest configuration. After operations of increasing or decreasing the capacity of a PD node, there should be two existing nodes as the members of the PD cluster before the operations to avoid the stale PD node list. If this condition is not met, perform the PD transfer leader operation manually to update the PD node list.
```

Comment:
```suggestion
> The PD Client in TiKV caches the list of PD nodes. Currently, the list is updated only if the PD leader is switched or the TiKV server is restarted to load the latest configuration. To avoid TiKV caching an outdated list, there should be at least two existing PD members in the PD cluster after increasing or decreasing the capacity of a PD node. If this condition is not met, transfer the PD leader manually to update the list of PD nodes.
```